rasa_core/policies/ensemble.py

```diff
@@ -217,10 +217,17 @@ def load(cls, path: Text) -> 'PolicyEnsemble':
 
     @classmethod
     def from_dict(cls, dictionary: Dict[Text, Any]) -> List[Policy]:
+        policies = dictionary.get('policies')
+        if policies is None:
+            raise InvalidPolicyConfig("The policy configuration file has to "
+                                      "include a key 'policies'.")
```

Comment:
Maybe make this a little easier to understand with e.g.  "You didn't define any policies, please define them under `policies:` in your yaml file"